facade-app/src/Components/Header.tsx

```diff
@@ -36,43 +41,53 @@ export class Header extends React.Component<Props> {
               <RSuiteNav.Item
                 onSelect={() =>
                   setGlobalState({
+                    ...globalState,
                     activateHomePage: true,
                     activateAboutUsPage: false,
                   })
                 }
-                icon={<RSuiteIcon icon="home" />}
+                icon={<RSuiteIcon icon="home" role={"img"} />}
               >
                 Home
               </RSuiteNav.Item>
               <RSuiteNav.Item
                 onSelect={() =>
-                  setGlobalState(
-                    { activateHomePage: false, activateAboutUsPage: true },
-                    true
-                  )
+                  setGlobalState({
```

Comment:
@Dripcoding  please put the `...globalState` in line 30 as well. You can do that in next PR.